notebooks/02_analyses/hatesonar_analysis.ipynb

```diff
@@ -0,0 +1,1416 @@
+{
```

Comment:
<p>you can add the following to the top of the notebook to hide these warnings:</p><p> </p><pre class="ql-syntax" spellcheck="false">import warnings
warnings.filterwarnings('ignore')
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/mailing-list-analysis-toolkit/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>